generate-docker-image-tags/action.yml

```
@@ -62,14 +62,17 @@ runs:
 
         if [ "$GITHUB_EVENT_NAME" == "pull_request" ]
         then
-          PR_NUMBER=${{ github.event.number }}
-        else
+          PR_NUMBER="${{ github.event.number }}""
```

This look like a typo.
```suggestion
          PR_NUMBER="${{ github.event.number }}"
```